fhir-server-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -18,30 +18,6 @@
         <url-pattern>/*</url-pattern>
     </servlet-mapping>
 
-    <!-- Security definitions for the web app -->
-    <security-role>
-        <role-name>FHIRUsers</role-name>
-    </security-role>
-    <security-constraint>
-        <web-resource-collection>
-            <web-resource-name>FHIR REST API</web-resource-name>
-            <url-pattern>/*</url-pattern>
-            <http-method>GET</http-method>
-            <http-method>POST</http-method>
-            <http-method>PUT</http-method>
-            <http-method>PATCH</http-method>
-            <http-method>DELETE</http-method>
-            <http-method>HEAD</http-method>
-            <http-method>TRACE</http-method>
```

Comment:
This is pretty explicit whitelisting.  What happens with websockets? What happens with OPTIONS?  are there any behavior changes?

Reply:
Websockets seems to work the same (with the default config at least).  OPTIONS also worked the same (always open), but had some interesting behavior when I added the mpJwt config:  it still returned HTTP 200 and listed the allowed methods, but it also included a `WWW-Authenticate` header with value `Bearer realm="MP-JWT", error="invalid_token"` when I invoked it without a jwt.